docs/mllib-evaluation-metrics.md

```diff
@@ -182,17 +183,22 @@ $$\hat{\delta}(x) = \begin{cases}1 & \text{if $x = 0$}, \\ 0 & \text{otherwise}.
       </td>
     </tr>
     <tr>
-      <td>Overall Precision</td>
+      <td>Micro Accuracy</td>
+      <td>$ACC = \frac{TP}{TP + FP} = \frac{1}{N}\sum_{i=0}^{N-1} \hat{\delta}\left(\hat{\mathbf{y}}_i -
+        \mathbf{y}_i\right)$</td>
+    </tr>
+    <tr>
+      <td>Micro Precision</td>
```

Comment:
Nit: I might call this "(Micro) Precision" to deemphasize the modifier. I personally had never heard it, and it may be a sklearn-specific description (?)  So maybe this makes it more readily understandable as "yes, just the definition you probably expect"